assets/templates/partials/release/contents/about-the-data.tmpl

```diff
@@ -6,7 +6,7 @@
         alt="Census 2021 logo"
         class="ons-u-mb-xs"
       >
-      <p>This release includes data from Census 2021.</p>
+      <p>{{- localise "CensusPhrasingNationalStatistic" $.Language 1 -}}</p>
```

Comment:
This choice of key `CensusPhrasingNationalStatistic` risks confusing the census with the national statistic designation. I'd suggest replacing it with `AboutDataCensus` instead.